### PR TITLE
Karimh

### PR DESCRIPTION
**Pull request by karimhchitrali**
Added a background and made my self-driving car!

### CODE REVIEW

File: hw03_karimh.py [hunk 1]
```diff
@@ -8,20 +8,111 @@
 
 import turtle
 
-def function_1():
-    "include docstring"
+#Assigning name to the turtle
+Heer = turtle.Turtle()
```
Comment (sheggen) [inline hw03_karimh.py:12]: There should not be code outside of `main()`. This likely needs to be inside main, and passed into functions as parameters.

File: hw03_karimh.py [hunk 2]
```diff
@@ -8,20 +8,111 @@
...
 
-def fucntion_4():
-    "docstring"
+    Heer.fillcolor("black")
+    Heer.begin_fill()
+
+    Heer.pensize(th)
+    Heer.circle(rad)
+    Heer.end_fill()
+def lower_body(l, w):
```
Comment (sheggen) [inline hw03_karimh.py:37]: Check PEP8 warning; functions should have two blank lines before them.

File: hw03_karimh.py [hunk 3]
```diff
@@ -8,20 +8,111 @@
...
+    wn.bgcolor("lightblue")
+    Heer.speed(0)
+
+    #Moves the turtle head from the center to the porper starting place
+    Heer.penup()
+    Heer.goto(-150, -200)
+    Heer.pendown()
+
     #call functions
+    tyre_1(50, 5)   #calls function 1 (making tyre 1)
```
Comment (sheggen) [inline hw03_karimh.py:89]: After `Heer` is moved to `main()`,  you probably need to pass it to this function call (and catch it in the function definition above).

File: hw03_karimh.py (hunk 2 above)
Comment (karimhchitrali) [reply hw03_karimh.py:37]: Resolved all PEP8 warnings, added a space between # and the first work of the comment and added two blank lines before functions.

File: hw03_karimh.py (hunk 3 above)
Comment (karimhchitrali) [reply hw03_karimh.py:89]: I added a parameter (H) for Heer in all my functions and called it in main.

**Comment by karimhchitrali** [comment]
Made changes for the function calls as pointed out by Dr. Heggen.

File: hw03_karimh.py (hunk 1 above)
Comment (sheggen) [reply hw03_karimh.py:13]: `Heer` is still outside of the functions.

File: hw03_karimh.py [hunk 4]
```diff
@@ -6,22 +6,122 @@
...
-def function_1():
-    "include docstring"
+# Assigning name to the turtle
+Heer = turtle.Turtle()
+
+
+def tyre_1(rad, th, H):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+    "Makes the left tyre of the car"
```
Comment (sheggen) [inline hw03_karimh.py:19]: Docstrings have a specific format: 

```
'''
Docstring format
'''
```

File: hw03_karimh.py [hunk 5]
```diff
@@ -6,22 +6,122 @@
...
 
 def main ():
-    #call functions
+    # Creates method for the screen
+    wn = turtle.Screen()
+    wn.bgpic("chitral-valley.gif")     # Adds a background picture
+
+    wn.colormode(255)
+    wn.bgcolor("lightblue")
+    Heer.speed(0)
```
Comment (sheggen) [inline hw03_karimh.py:89]: `Heer = turtle.Turtle()` should go here

File: hw03_karimh.py (hunk 4 above)
Comment (karimhchitrali) [reply hw03_karimh.py:19]: added 3 double-quotations to fix doctring format.

File: hw03_karimh.py (hunk 5 above)
Comment (karimhchitrali) [reply hw03_karimh.py:89]: Moved Heer = turtle.Turtle() before Heer.speed(0))

**Pull request merged**